src/docs/truffle/reference/configuration.md

```diff
@@ -124,6 +124,22 @@ For each network, you can specify `host` / `port`, `url`, or `provider`, but not
 - https://sandbox.truffleteams.com/yoursandboxid
 - wss://sandbox.truffleteams.com/yoursandboxid
 
+### console
+
+Use this option to configure the console environment. This is the environment you get when using `truffle console` or `truffle develop`.
+
+* `require`: Use this to specify JavaScript that will be loaded into the console environment at startup. Your JavaScript's default
+export must be an object with named keys. Each key will used as a variable name, assigned its value, and be available in the console environment.
+For example, if your JavaScript is `module.exports = { myVariable: "three doughnuts please!" }` then you will have the variable `myVariable`
+available in the console with the value `"three doughnuts please!"`.
+
+You may specify this option in several ways.
+1. The value of require can be a string path to the target JavaScript file to be loaded.
+2. An object with a `path` property that is a string path to the JavaScript to be loaded.
+3. An object with a `path` property that is a string path to the JavaScript to be loaded as well as an `as` property
```

Comment:
Is it possible to add code snippets for each option? I think this will enhance the docs and add something a user could cut and paste into their own config.